src/snakeoil/chksum/defaults.py

```diff
@@ -13,15 +13,16 @@
 
 blocksize = 2 ** 17
 
-sha1_size = 40
+blake2b_size = 128
+blake2s_size = 64
 md5_size = 32
 rmd160_size = 40
+sha1_size = 40
 sha256_size = 64
 sha512_size = 128
 sha3_256_size = 64
 sha3_512_size = 128
-blake2b_size = 128
-blake2s_size = 64
+whirlpool_size = 128
```

Comment:
Hmm, I'm wondering if there's a point in keeping these variables rather than inlining the numbers in the list below.

Reply:
I'd avoid refactoring of that sort at this stage.  The constants were broken out because the implementation could fallback to alternative sources for CHF- whirlpool for example, isn't guaranteed by hashlib.  That capability still has some value (whirlpool for example).

Either way, I deal w/ repositories that use whirlpool, so I'm more interested in restoring pkgcore as usable- and able to be developed- then cleaning code at this stage.  The refactoring will follow, but that's not going to happen if I can't get pkgcore back to a state where I can use it daily.